packages/headless/src/components/toast/toast.tsx

```diff
@@ -0,0 +1,32 @@
+import { component$ } from '@builder.io/qwik';
+interface ToastProps {
+  value?: string;
+  /**
+   * The controlled state of the toast.
+   */
+
+  label?: string;
+  alert?: string;
+  /**
+   * The state of the toast when initially rendered. Use `defaultPressed`
+   * if you do not need to control the state of the toast.
+   * @defaultValue false
+   */
+}
+
+export const Toast = component$((props: ToastProps) => {
+  const { alert = 'info', label, ...toastProps } = props;
+
+  return (
+    <div class="toast" style={{ position: 'absolute' }} {...toastProps}>
+      <div
+        class={`alert alert-${alert}`}
```

Comment:
Who defines alert class?

Reply:
![image](https://user-images.githubusercontent.com/37772742/219448751-c758b63f-e1dc-42fd-b5b9-a763dae9c084.png)
in my localhost it looks like this but I recognize that every toast will be in black background but not sure how to change it